core/src/com/mygdx/game/models/Session.java

```diff
@@ -0,0 +1,209 @@
+package com.mygdx.game.models;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Random;
+
+/**
+ * The Session model keeps track of players, selected brains, total brains and
+ * rounds.
+ *
+ * This class implements the MVC pattern.
+ * The parameters given in the constructor makes it easier to implement the quality requirement M1.
+ */
+
+public class Session {
+    public static final int MAX = 9999;
+    public static final int MIN = 1000;
+
+    private final int maxHitPoints;
+    private final int brainDamage;
+    private final int maxSelectedBrains;
+    private final int maxRounds;
+    private final ArrayList<Player> players;
```

Comment:
I'm hesitant to have any of our models keep track of the players, since this will make firebase much more complicated and will require way more synchronisation. If the game were just running on one phone, then this would be the solution, but since we're using a realtime database, this means that firebase will have to be updated everytime a player does something. 

One way to do this is that the host keeps track of everyone, but the other players do not. But this will require atleast one new class.